### PR TITLE
Add fastconformer model for decoder timestamp

### PR DESCRIPTION
**Pull request by biscayan**
# What does this PR do ?

Add fastconformer model for decoder timestamp.
- Fastconformer model shoud have stride in 0.08 secs
- You can use fastconformer model for diarization in offline_diar_with_asr_infer.py 

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Jenkins CI
To run Jenkins, a NeMo User with write access must comment `jenkins` on the PR.

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to #2857

### CODE REVIEW

**Review by titu1994 — COMMENTED** [review]
Thanks ! @tango4j for review

File: nemo/collections/asr/parts/utils/decoder_timestamps_utils.py [hunk 1]
```diff
@@ -348,6 +348,17 @@ def set_asr_model(self):
...
+        elif 'fastconformer' in self.ASR_model_name.lower():
+            self.run_ASR = self.run_ASR_BPE_CTC
+            self.encdec_class = EncDecCTCModelBPE
+            self.decoder_delay_in_sec = if_none_get_default(self.params['decoder_delay_in_sec'], 0.08)
+            self.word_ts_anchor_offset = if_none_get_default(self.params['word_ts_anchor_offset'], 0.12)
+            self.asr_batch_size = if_none_get_default(self.params['asr_batch_size'], 16)
+            self.model_stride_in_secs = 0.08
+            # fastConformer requires buffered inference and the parameters for buffered processing.
+            self.chunk_len_in_sec = 5
+            self.total_buffer_in_secs = 25
```
Comment (nithinraok) [inline nemo/collections/asr/parts/utils/decoder_timestamps_utils.py:360]: Thanks for the PR, have you experimented with these values?

Comment (nithinraok) [inline nemo/collections/asr/parts/utils/decoder_timestamps_utils.py:360]: Thanks for the PR, have you experimented with these values?

**Comment by tango4j** [comment]
Let us run on our test samples and make sure the timestamps are working smoothly.
Thank you for uploading this PR.

File: nemo/collections/asr/parts/utils/decoder_timestamps_utils.py (hunk 1 above)
Comment (biscayan) [reply nemo/collections/asr/parts/utils/decoder_timestamps_utils.py:360]: No, I didn't change these values, I only have experimented with the stride seconds

**Review by nithinraok — DISMISSED** [review]
LGTM. @tango4j for final signoff.

**Comment by nithinraok** [comment]
@biscayan pls follow steps listed at https://github.com/NVIDIA/NeMo/pull/8341/checks?check_run_id=21297714430 to sign off your commit

**Pull request closed without merging**